src/audacity.desktop.in

```diff
@@ -53,4 +53,4 @@ Categories=AudioVideo;Audio;AudioVideoEditing;
 Exec=env UBUNTU_MENUPROXY=0 @AUDACITY_NAME@ %F
 StartupNotify=false
 Terminal=false
-MimeType=application/x-audacity-project;@MIMETYPES@
+MimeType=@MIMETYPES@;
```

Comment:
`application/x-audacity-project` has been moved to the CMake file to ensure that `@MIMETYPES@` is never empty. This ensures we never get zero or two semicolons at the end of this line, which would cause the AppImage packaging tools to complain about a badly formed list variable in the `.desktop` file.